lib/commands/actions.js

```diff
@@ -237,6 +239,78 @@ function verifyOptions (opts) {
   return {bundleId: opts.bundleId};
 }
 
+commands.mobileInstallApp = async function (opts = {}) {
+  const {app} = opts;
+  if (_.isEmpty(app)) {
+    log.errorAndThrow(`'app' parameter is mandatory. '${app}' is given instead.`);
+  }
+
+  let dstPath = app;
```

Comment:
All this logic is in [configureApp](https://github.com/appium/appium-base-driver/blob/master/lib/basedriver/helpers.js#L14), which is available in this driver as `this.helpers.configureApp`.

Reply:
I've improved it a bit to make sure all the intermediate files are cleaned up after the app is installed

Reply:
I'd rather see those improvements made to the central place and not duplicate all of this over here.

Reply:
ok, point taken